### ISSUE TITLE
Move jest config into package.json

### ISSUE THREAD

**Issue by ahutchings** [tommikaikkonen/redux-orm#166]
Having the jest config in package.json is handy when doing things with jest that don't already have scripts defined in package.json. For example, debugging or profiling unit tests using `node --inspect node_modules/.bin/jest --runInBand` as documented [here](http://facebook.github.io/jest/docs/en/troubleshooting.html). With the config in package.json, the extra --config flag is not needed. 

Thoughts? I can submit a PR if there are no objections.

cc @plandem

**Comment by tiii** [comment]
I prefer the storage for approach. To prevent package.json from getting too big. You can always add a test:debug task or something similar if you want.

**Comment by ahutchings** [comment]
Looks like jest will now automatically use a [`jest.config.js`](https://facebook.github.io/jest/docs/en/configuration.html) file if it exists. I've tested this locally by renaming `jest.conf.json` to `jest.config.js` and exporting the json with `module.exports =`. This seems like the ideal way to solve this issue since it allows for debugging using the snippet I posted while still keeping the config out of package.json.

I'll submit a PR if this sounds okay.
